tests/test_utils/logging_setup.py

```diff
@@ -28,7 +28,7 @@ class ToTwistedHandler(logging.Handler):
     def emit(self, record):
         log_entry = self.format(record)
         log_level = record.levelname.lower().replace("warning", "warn")
-        self.tx_log.emit(
+        self.tx_log.emit(  # type: ignore
```

Comment:
For some reason mypy things that `tx_log` here is `object`, I'm not really sure why though... Anyway, it complains that object has no `emit` method.

Reply:
I suspect this is fixed by twisted/twisted#1541?